experimental/traffic-portal/src/app/api/physical-location.service.ts

```diff
@@ -23,33 +22,75 @@ import { APIService } from "./base-api.service";
  */
 @Injectable()
 export class PhysicalLocationService extends APIService {
-	public async getPhysicalLocations(idOrName: number | string): Promise<PhysicalLocation>;
-	public async getPhysicalLocations(): Promise<Array<PhysicalLocation>>;
+	public async getPhysicalLocations(): Promise<Array<ResponsePhysicalLocation>>;
+	public async getPhysicalLocations(nameOrID: string | number): Promise<ResponsePhysicalLocation>;
+
 	/**
-	 * Gets one or all PhysicalLocations from Traffic Ops
+	 * Gets an array of physicalLocations from Traffic Ops.
```

Comment:
The object is `[Response]PhysicalLocation` and the concept it represents is a "Physical Location," either of which is more appropriate than `physicalLocation`.